.npmignore

```diff
@@ -10,5 +10,7 @@ src
 test
 tools
 .vscode/
+build/*
```

Comment:
Note: double-check `npm pack --dry-run` to ensure we don't end up with an empty `build` directory in there.

Reply:
Will do

In other comments, it was mentioned that there needs to be a copy in `docs`.  There currently isn't.  Does it go under the standard name, under pseudo-root?

Reply:
https://peggyjs.org/vendor/peggy/peggy.min.js is served from `/docs/vendor/peggy/peggy.min.js` currently

Reply:
Second distro target added.  Have not yet dry run; further work is needed here